modules/react/popup/lib/hooks/usePopupCloseButton.ts

```diff
@@ -2,8 +2,9 @@ import {createElemPropsHook} from '@workday/canvas-kit-react/common';
 import {usePopupModel} from './usePopupModel';
 
 /**
- * Adds the necessary props to a close button component. Used by the `Popup.CloseButton`
- * subcomponent and `Popup.CloseIcon` subcomponent.
+ * Adds the necessary props to a close button components. Used by the
```

Comment:
A pluralization was added that isn't right:
```suggestion
 * Adds the necessary props to a close button component. Used by the
```